runtime/data/script/kernel/serial.lua

```diff
@@ -19,7 +19,10 @@ local function resolve_handles(data, seen)
    end
 end
 
-Serial.save = serpent.dump
+Serial.save = function(s)
```

Comment:
This changes something, or is just refactoring for future extension?

Reply:
I found I needed to use `inspect` often enough to for debugging so I made it easier to insert when needed.